### ISSUE TITLE
accessibility id is not visible when inspect element

### ISSUE THREAD

**Issue by AlirezaAzizi145** [appium/appium#15905]
Hi, i use **Appium** in my **Nativescript** project. i want to find elements by `accessibility-id` but that's not visible.
i dont want to use `xpath` instead `accessibility-id` beacause i have a lot of changes on my UI, so it's not work for me.
There is an interesting point that the `accessibility-id` is only show when i inspect my **Nativescript searchbar**

**Comment by mykola-mokhnach** [comment]
Please fill out the issue report properly and provide the necessary details.

Usually all values that are visible in the page source XML are taken from the accessibility framework and this does not depend on appium but rather on to what your app exposes to the accessibility.

**Comment by mykola-mokhnach** [comment]
Closed because of no response
